widgets/views/templateSelectModal.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+use humhub\widgets\Button;
+use humhub\widgets\Modal;
+use humhub\widgets\ModalButton;
+use yii\helpers\Html;
+use humhub\modules\wiki\helpers\Url;
+use yii\widgets\ActiveForm;
+
+$templateOptions = [];
+
+foreach ($templates as $template) {
+    $templateOptions[$template->id] = Html::encode($template->title);
+}
+
+?>
+
+<?= Modal::widget([
+    'id' => 'templateSelectModal',
+    'header' => '<strong>New page</strong>',
+    'body' => '<div class="text-center">' .Yii::t('WikiModule.base', '<p>Start with an empty page or template<p>') .
+                Button::primary(Yii::t('WikiModule.base', 'Create blank page'))->id('blankPageBtn')->loader(false) .
```

Comment:
Sometimes the button seem not to work.
Demo:
[blank_page_button.webm](https://github.com/user-attachments/assets/07d324b2-90c4-48bf-ad09-e53956da53d9)